src/Appwrite/Extend/Exception.php

```diff
@@ -164,6 +171,18 @@ class Exception extends \Exception
     /** Webhooks */
     public const WEBHOOK_NOT_FOUND                 = 'webhook_not_found';
 
+    /** Router */
+    public const ROUTER_UNKNOWN_HOST                = 'router_unknown_host';
+    public const ROUTER_INVALID_TYPE                = 'router_unknown_type';
```

Comment:
This error can't really happen, can we just throw 500?

Reply:
"ROUTER_UNKNOWN_HOST" means you are making request from domain which appwrite doesnt know. So by default we dont trust it. Very common problem. Let'ß say you point DNS of `*.matejbaco.eu` to Appwrite.. And then I visit `blabla.matejbaco.eu`. It should tell me it's untrusted. Thats purpose of the error.

Error says `Host is not trusted. Add a custom domain to your project first.`

---

ROUTER_INVALID_TYPE shuold never happen, correct. Removing it and making it server error.